src/app/ui/portfolio/PortfolioHeaderBreakdown.tsx

```diff
@@ -63,6 +70,35 @@ export const PortfolioHeaderBreakdown = () => {
     return formattedPositions;
   }, [formattedPositions, portfolioWelcomeScreenClosed]);
 
+  useEffect(() => {
+    if (
+      isLoading ||
+      portfolioHasBeenTracked.current ||
+      !portfolioWelcomeScreenClosed
+    ) {
+      return;
+    }
+    portfolioHasBeenTracked.current = true;
+    const trackingData = parseEarnPortfolioDataToTrackingData(
+      connectedAddresses,
+      tokens,
+      defiPositionGroups,
+    );
+    trackEvent({
+      category: TrackingCategory.Portfolio,
+      action: TrackingAction.PortfolioPageOverview,
+      label: 'portfolio_page_overview',
+      data: trackingData,
+    });
```

Comment:
extract to something like `trackPortfolioEvent`, so the effect is focused on state management


```suggestion
    trackPortfolioEvent(
      connectedAddresses,
      tokens,
      defiPositionGroups,
    )
```